sdk/documentintelligence/ai-document-intelligence-rest/CHANGELOG.md

```diff
@@ -5,23 +5,98 @@
 ### Features Added
 
 - Exports method `streamToUint8Array` to support converting a `NodeJS.ReadableStream` to a `Uint8Array`. This is necessary to read the pdf and png responses from the results of an analysis.
```

Comment:
```suggestion
- Adds `streamToUint8Array`, a convenience function that buffers a `NodeJS.ReadableStream` in a `Uint8Array`. It can be used to read the pdf and png responses from the results of an analysis.
```

Reply:
I wonder why can't we stream the response directly to disk instead of buffering it first in memory?

Reply:
This function is to mainly being served from this package to close the gap of not supporting `Uint8Array` return type though it is being offered.